diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -1150,6 +1173,40 @@ fun doesLambdaContainIt(lambdaNode: ASTNode): Boolean {
     return hasNoParameters(lambdaNode) && hasIt
 }
 
+/**
+ * Checks that node (property or backing field) has pair node (backing field or property)
+ *
+ * @return true if node has a pair
+ */
+@Suppress("CyclomaticComplexMethod")
+internal fun isPairPropertyBackingField(propertyNode: ASTNode?, backingFieldNode: ASTNode?): Boolean {
```

Comment:
please document `@params`, ie `propertyNode` and `backingFieldNode`

Reply:
Done